src/main/java/uk/gov/companieshouse/company_appointments/model/view/CompanyAppointmentFullRecordView.java

```diff
@@ -169,6 +180,12 @@ public Boolean getIsPre1992Appointment() {
         return isPre1992Appointment;
     }
 
+    public ContactDetails getContactDetails() {return contactDetails;}
```

Comment:
This should be on three lines like the other getters
```suggestion
    public ContactDetails getContactDetails() {return contactDetails;}
```